vault/dendron.tutorial.rich-formatting.md

```diff
@@ -16,22 +16,21 @@ Dendron supports an extended Markdown syntax, which provides a lot of options fo
 |Bold|**Hello World**|
```

Comment:
folks looking on the website won't be able to see the markdown. add a code block of the syntax so that users can see both the raw markdown and the formatted one

Reply:
tables can be formatted nicely in markdown as well. would aligh it so it looks nice even in the editor